### ISSUE TITLE
a11y: 3 tab stops for every book, all leading to same place (plus bad link text)

### ISSUE THREAD

**Issue by dstorey** [smashingmagazine/redesign#107]
https://next.smashingmagazine.com/printed-books/

Steps:

Tab to and past the first book on the page.

Expected:

As all links for the books (image, title, and read more) all go to the same place, I'd expect only one tab stop

Actual:

3 for every link, making it more frustrating to navigate the page. Futher more the image and read more link have weak link text, making it harder to navigate with screen reader too (no alt text, and "learn more -> respectively)

This fails WCAG due to:

"When adjacent links go to the same location (such as a linked product image and an adjacent linked product name that go to the same product page) this results in additional navigation and repetition for keyboard and screen reader users."

The lack of alt text and context to read more is additionally part of WCAG criteria 2.4.4

Additionally the tab order is kind of strange as it tabs first to the image down the right side of the block, then back up again to the title, then the read more.

How I often solve these issues is by pinning and stretching the link/button to the full size of the element, such as:

* remove anchor from around the image, and set alt=""
* remove the anchor from inside the H3
* set position relative on the containing element
* set position: absolute, top, left, right, and bottom; 0 for the a::before (using whatever class it uses)

This way the before will fill the entire containing element and be clickable with only one tab stop. In this case the area is fairly big so not sure if you'd want it all to be clickable. 

If you do this you need to make the learn more anchor have acceptable accessible name. You can for example use aria-label="learn more about [[title of book]]" or use aria-labelledby to point to the ID of the h3.

Alternatively I'd put the alt="" on the image again and wrap the anchor around both the image and the h3. This will give one tab stop and the correct accessible name. That won't solve the learn more button being a duplicate though.

**Comment by vitalyfriedman** [comment]
Thanks @dstorey — will be working on it!
